sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/EventHubScope.cs

```diff
@@ -34,7 +34,7 @@ public sealed class EventHubScope : IAsyncDisposable
         private const double RetryExponentialBackoffSeconds = 2.0;
 
         /// <summary>The number of seconds to use as the basis for applying jitter to retry back-off calculations.</summary>
-        private const double RetryBaseJitterSeconds = 8.0;
+        private const double RetryBaseJitterSeconds = 10.0;
```

Comment:
Bringing this into alignment with the change that I made in Track One.  It seems to have help settled clustered failures from the management plane.